app/services/exports/export_donations_csv_service.rb

```diff
@@ -0,0 +1,112 @@
+module Exports
+  class ExportDonationsCSVService
+    def initialize(donations)
+      # TODO: remove this comment unless we actually do some kind of
+      # clever lookup
+      #
+      # Use a where lookup so that I can eager load all the resources
+      # needed rather than depending on external code to do it for me.
+      # This makes this code more self contained and efficient!
+      @donations = donations
```

Comment:
You should be able to eager load using the same approach as:

https://github.com/rubyforgood/human-essentials/blob/416ea1edec75b7bd462c3e74496a265176be1abb/app/services/exports/export_distributions_csv_service.rb#L7